knowledge/geography/islands/prince_island/qna.yaml

```diff
@@ -0,0 +1,83 @@
+    most of the island.
+  question: How high are the cliffs surrounding Prince Leopold Island?
+- answer: The island and Somerset Island to its south were first sighted by a
+    European on August 4, 1819, during William Parry's 1819 expedition.
+  question: Prince Leopold Island was first sighted during which expedition?
+- answer: Large numbers of arctic seabirds, thick-billed murres, northern fulmars,
+    black-legged kittiwakes, and black guillemots breed on the cliff ledges, arriving
+    in the vicinity in May or early June and departing by mid-September.
+  question: What animals live on Prince Leopold Island?
+- answer: "74°N 90°W"
+  question: What are the coordinates of Prince Leopold Island?
+- answer: It is located in Lancaster Sound at the junction of Prince Regent
+    Inlet and Barrow Strait.
+  question: What body of water surrounds Prince Leopold Island?
+- answer: Prince Leopold Island is a significant breeding ground for large
+    populations of several arctic bird species.
+  question: For what species of animals is Prince Leopold Island a significant
+    summer habitat for?
+- answer: The majority of the island's area of 68 km2 (26 sq mi) is occupied by a
+    plateau generally ranging in altitude between around 240 m (790 ft) and 300 m
+    (980 ft).
+  question: What is the altitude of the plateau that makes up most of Prince
+    Leopold Island?
+- answer: Vegetation consists of forbs, lichens, and mosses, though the top of the
+    island is mostly devoid of plants. Denser concentrations of vegetation occur in
+    seepage areas on the spits, including sedges and some dwarf willow
+  question: What does the flora consist of on Prince Leopold Island?
+- answer: The majority of the island's area of 68 km2 (26 sq mi) is occupied by a
+    plateau generally ranging in altitude between around 240 m (790 ft) and 300 m
+    (980 ft).
+  question: What is the topography of Prince Leopold Island?
+- answer: Somerset Island
+  question: What is the name of the island to the south of Prince Leopold Island?
+- answer: The plateau is divided in the east by a gorge that runs north-south
+    across the width of the island, while smaller steep canyons skirt the edge
+    of the island and main gorge at various points. Four gravel spits, in the
+    north, southwest, south, and southeast of the island, jut out from the coast.
+    During the summer, surface water on the island is scarce. A few small shallow
+    lakes exist on the plateau; the largest, Far Lake, is about 0.5 km (0.31 mi)
+    long, and is located near the eastern cliffs
+  question: What is the terrain like on Prince Leopold Island?
+- answer: Large numbers of arctic seabirds, thick-billed murres, northern fulmars,
+    black-legged kittiwakes, and black guillemots breed on the cliff ledges, arriving
+    in the vicinity in May or early June and departing by mid-September.
+  question: What types of birds live on Prince Leopold Island?
+- answer: 306 m (1004 ft)
```

Comment:
This information is missing from the Markdown version

https://github.com/instructlab/taxonomy/pull/1154#issuecomment-2146437322

Reply:
@juliadenham updated the Markdown in commit 706f365a256950f408707e2f03fe76e380c734c6